canary/apps/react/cra-ts/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@aws-amplify/ui-react": "^2.2.1",
+    "@aws-amplify/ui-react": "latest",
```

Comment:
Should this be v3 latest? It seems running against latest on v3 branch is meaningless

Reply:
Yep, updating.